BlueBrain/Search
# Make entrypoint startup faster

Fixes #411.

## Description
Our CLI entrypoints have a very slow startup time. This is because all entrypoint submodules are imported in `bluesearch.entrypoint.__init__`.

The problem is solved by removing these imports.

## How to test?

- `pytest tests/test_entrypoint`
- Try running the entrypoint in the shell, observe much faster startup time.

## Checklist

  (if it is not the case, please create an issue first).
  (if needed)
  (if needed)
  (if needed)
  (if a function is added or modified)
